Source/CheckoutTheme.swift

```diff
@@ -1,8 +1,38 @@
 import Foundation
 import UIKit
 
+
 /// Class used to customize the views.
 public final class CheckoutTheme {
```

Comment:
We should not be using CheckoutTheme for padding, as it has old code as well. either we clean up the CheckoutTheme or create something new for padding enum.